### ISSUE TITLE
Copy paste is not working with OpenRouter Deepseek-coder chat field

### ISSUE THREAD

**Issue by arttus** [saoudrizwan/claude-dev#153]
For some reason, copy/paste is not working with OpenRouter Deepseek-coder. If I switch to another model I can paste into the chat window.

**Comment by soulforsoup** [comment]
Same here. Mac OS.

**Comment by saoudrizwan** [comment]
Is this happening after/during a CLI command is executed? I just ran into this issue myself, but pasting works otherwise.

**Comment by arttus** [comment]
I only see the issue on Deepseek, other models C/P works fine.

**Comment by jginstitute** [comment]
> Is this happening after/during a CLI command is executed? I just ran into this issue myself, but pasting works otherwise.

I just installed ClaudeDev for the first time ever, put in my OpenRouter key, picked Deepseek, and was immediately faced with the same issue (can't paste into chat). After I went to settings and changed the model to Sonnet, paste worked.

**After some testing, I found that the Mistral model also suffers from the same issue!**

So the current workaround is to change the model to anything other than Deepseek or Mistral, perform the paste, and then set the model back to Deepseek or Mistral.
